### ISSUE TITLE
Revision statistics

### ISSUE THREAD

**Issue by shal** [opencars/wanted#1]
We should list aggregated revisions by month for last year.

```sh
http GET https://api.opencars.pp.ua/wanted/revisions/stats
```

```JSON
[
  {
    "month": "January",
    "added": 10,
    "removed": 20
  },
  { 
    "month": "February",
    "added": 100,
    "removed": 200
  },
  { 
    "month": "March",
    "added": 2,
    "removed": 1
  }
]
```

**Comment by shal** [comment]
Closed in  65b093a

**Comment by shal** [comment]
Closed via 448e5f1
